Composer/packages/client/src/components/AddRemoteSkillModal/CreateSkillModal.tsx

```diff
@@ -375,17 +372,6 @@ export const CreateSkillModal: React.FC<CreateSkillModalProps> = (props) => {
           </Fragment>
         )}
       </DialogWrapper>
-      {showAuthDialog && (
```

Comment:
This no longer shows the auth dialog.  Does it need to require authentication through the new auth?

Reply:
that authentication now happens automatically when the user gets to the publish profile dialog. the auth dialog is part of the global component now, so doesn't have to be invoked explicitly here.